lua/libmodal/src/utils/api.lua

```diff
@@ -5,8 +5,28 @@
 --]]
 
 local api = {}
+local globals = require('libmodal/src/globals')
 local vim_api = vim.api
 
+---------------------------------
+--[[ SUMMARY:
+	* Send a character to exit a mode.
+]]
+--[[ PARAMS:
+	* `exit_char` => the character used to exit the mode, or ESCAPE if none was provided.
+]]
+---------------------------------
+function api.mode_exit(exit_char)
```

Comment:
New function to exit modes. It works by sending an exit character to the input buffer.